File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewModelMyTreeViewElementsRow.cs
/// <filename>
///     SelectRowCommand_ViewModelMyTreeViewElementsRow.cs
/// </filename>


namespace commands
{
    public class SelectRowCommand_ViewModelMyTreeViewElementsRow
  {
        private string columnCellLabelText;
     
        public     virtual string getColumnCellLabelText()
    {
      return this.columnCellLabelText;
    }
     
        private int RowIndex;
     
        public     virtual int getRowIndex()
    {
      return this.RowIndex;
    }
     
        private int RowDepth;
     
        public     virtual int getRowDepth()
    {
      return this.RowDepth;
    }
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewModelMyTreeViewElementsRow.cpp
#include "SelectRowCommand_ViewModelMyTreeViewElementsRow.h"
#include <string>

namespace commands
{
  std::string SelectRowCommand_ViewModelMyTreeViewElementsRow::getColumnCellLabelText()
  {
    return this->columnCellLabelText;
  }
  int SelectRowCommand_ViewModelMyTreeViewElementsRow::getRowIndex()
  {
    return this->RowIndex;
  }
  int SelectRowCommand_ViewModelMyTreeViewElementsRow::getRowDepth()
  {
    return this->RowDepth;
  }
}
